fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/JcrRdfTools.java

```diff
@@ -173,7 +174,7 @@ public Value createValue(final Node node,
      * @throws RepositoryException
      */
     public Value createValue(final ValueFactory valueFactory, final RDFNode data, final int type)
```

Comment:
I would think you could follow the pattern set by:
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/rdf/JcrRdfToolsTest.java#L312 or
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/rdf/JcrRdfToolsTest.java#L329